### PR TITLE
♻️  new About Section UI

### PR DESCRIPTION
**Pull request by MarioBatalha**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206412086603107

### CODE REVIEW

File: components/svg/plus.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { SVGProps } from './svg.types';
+
+const PlusSVG: FC<SVGProps> = ({ maxHeight, maxWidth, ...props }) => (
+  <svg
+    style={{ maxHeight, maxWidth }}
+    viewBox="0 0 24 24"
+    fill="none"
+    {...props}
+  >
+    <path d="M13 4V11H20V13H13V20H11V13H4V11H11V4H13Z" fill="black" />
```
Comment (atilasousa) [inline components/svg/plus.tsx:12]: currentColor?

File: views/home/about/index.tsx [hunk 2]
```diff
@@ -25,169 +25,126 @@ const AboutSection: FC = () => {
...
+              justifyContent="space-between"
+              alignItems="center"
+              mb="10rem"
             >
+              <Typography
+                fontFamily="Roboto"
+                fontSize="2rem"
+                fontStyle="normal"
+                fontWeight="400"
+                lineHeight="2.5px"
```
Comment (KipandaJr) [inline views/home/about/index.tsx:82]: `rem`instead `px`

**Review by KipandaJr — CHANGES_REQUESTED** [review]

**Review by git-marcopitra — APPROVED** [review]

File: views/home/about/index.tsx (hunk 2 above)
Comment (git-marcopitra) [reply views/home/about/index.tsx]: Not resolved @MarioBatalha

File: components/svg/plus.tsx (hunk 1 above)
Comment (git-marcopitra) [reply components/svg/plus.tsx]: Not Resolved @MarioBatalha

**Review by KipandaJr — APPROVED** [review]

**Review by git-marcopitra — APPROVED** [review]

**Review by git-marcopitra — APPROVED** [review]

**Pull request merged**